anubis-management-api/anubis/policies/routers.py

```diff
@@ -631,4 +666,21 @@ def delete_policy(
                 urllib.parse.quote(
                     db_policy.access_to)))
         logging.error(e)
+    opa_url = os.environ.get('OPA_ENDPOINT', 'http://127.0.0.1:8181')
```

Comment:
code below should be executed only if OPA_ENDPOINT is specified (leaving a way to use pull mode)